brooklyn-tosca-transformer/src/main/java/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaPlanToSpecTransformer.java

```diff
@@ -279,6 +278,32 @@ private String getParentId(NodeTemplate nodeTemplate) {
         return rootSpec;
     }
 
+    private void manageBrooklynPoliy(Map<String, EntitySpec<?>> specs,
+                                     NodeGroup group,
+                                     GenericPolicy policy) {
+
+        List<Location> foundLocations =null;
+        if(policy.getData().containsKey(GroupPolicyParser.VALUE)){
+            foundLocations = new BrooklynYamlLocationResolver(mgmt)
+                    .resolveLocations(ImmutableMap.of("location",
+                                    policy.getData().get(GroupPolicyParser.VALUE)), true);
+        } else {
+            Map<String, ?> data = MutableMap.copyOf(policy.getData());
+            data.remove("name");
```

Comment:
I remove name because for a policy, for example:
```
  groups:
    add_brooklyn_location:
      members: [ a_server ]
      policies:
        - brooklyn.location:
            jclouds:aws-ec2:
              region: us-west-2
              identity: user-key-id
              credential: user-key
```
`polocy.getData()` returns a map with two entries.
 - "jclouds:aws-ec2: contains a map with location properties.
 - "name": policy name (`brooklyn.location`).
`data` map will used by `BrooklynYamlLocationResolver` to find create the expected `Location` object.  

Then, I'd not like that the policy name will be found by `BrooklynYamlLocationResolver`.